src/V3Undriven.cpp

```diff
@@ -362,7 +408,14 @@ class UndrivenVisitor : public AstNVisitor {
                     UINFO(9, " Full bus.  Entryp=" << cvtToHex(entryp) << endl);
                     warnAlwCombOrder(nodep);
                 }
+                if (m_inContAssign && !m_constHasZ && entryp->isDrivenNonZ() &&
+                    entryp->isDrivenWhole()) {
+                    nodep->v3warn(MULTIDRIVEN,
+                                  "Multiple assignments to wire "
+                                  << nodep->prettyNameQ());
```

Comment:
Probably should also report to the other driver, though this will require a lot more tracking so maybe not worth it..

Reply:
I have added support for tracking the other driver. This is much nicer to debug with.